src/pgduckdb_planner.cpp

```diff
@@ -55,12 +56,23 @@ PlanQuery(Query *parse, ParamListInfo bound_params) {
 
 std::tuple<duckdb::unique_ptr<duckdb::PreparedStatement>, duckdb::unique_ptr<duckdb::Connection>>
 DuckdbPrepare(const Query *query, ParamListInfo bound_params) {
+
 	/*
 	 * Copy the query, so the original one is not modified by the
 	 * subquery_planner call that PlanQuery does.
 	 */
 	Query *copied_query = (Query *)copyObjectImpl(query);
+	/*
+	    Temporarily clear search_path so that the query will contain only fully qualified tables.
+	    If we don't do this tables are only fully-qualified if they are not part of the current search_path.
+	    NOTE: This still doesn't fully qualify tables in pg_catalog or temporary tables, for that we'd need to modify
+	   pgduckdb_pg_get_querydef
```

Comment:
super nitpick: it looks a bit weird to me that there are no `*`'s in front of every line of the comment.